deploy/app/modules/indexer/gateway.tf

```diff
@@ -142,4 +142,10 @@ resource "aws_route53_record" "api_gateway" {
     zone_id                = aws_apigatewayv2_domain_name.custom_domain.domain_name_configuration[0].hosted_zone_id
     evaluate_target_health = false
   }
-}
\ No newline at end of file
+
```

Comment:
is this all you need to do to get proper routing by region? That's kind of awesome.

Reply:
right? From what I read, they take periodic latency measurements and approximate where the request is coming from. I'm sure it won't be perfect, but I'm looking forward to seeing it in action.

Reply:
yea makes sense. I have an additional proposal after that for getting to optimum performance